src/lib/addons/index.ts

```diff
@@ -29,7 +29,7 @@ export const getAddons: (args: {
         new Webhook({ getLogger }),
         slackAddon,
         new TeamsAddon({ getLogger, unleashUrl }),
-        new DatadogAddon({ getLogger, unleashUrl }),
+        new DatadogAddon({ getLogger, unleashUrl }, { flagResolver }),
```

Comment:
I think you can simplify this with https://github.com/Unleash/unleash/pull/4752/files#r1328576551